pyuvsim/source.py

```diff
@@ -7,6 +7,8 @@
 import numpy as np
 import copy
 import sys
+import h5py
+import healpy as hp
```

Comment:
I'd suggest importing these only in the functions that need them to prevent adding more dependencies.

Reply:
fix this after we move the code to pyradiosky.